device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,285 @@
+{
+    "chassis": {
+        "name": "MSN2700",
```

Comment:
@keboliu: On master branch, `chassis.get_name()` returns "Mellanox-SN2700", not "MSN2700". Does the platform API need to be changed to return "MSN2700"?

Reply:
yes, need to change to "MSN2700"

Reply:
no change, json is correct, API needs to correct